### PR TITLE
Update Bridge Module to mint MOVE FA

### PR DESCRIPTION
**Pull request by 0xmovses**
# Summary
Creates a MOVE Fungible Asset that conforms the fungible asset standard. Updates the bridge modules to mint that asset. 

# Changelog
- Creates `MOVE.move`
- Updates counterparty.move and initiator.move to mint the MOVE FA

# Testing
- Tests running in CI
- To run locally : `cd protocol-units/bridge/move-modules && aptos move test`

# Outstanding issues
Follow up to PR to integrate the MOVE FA into the `module atomic_bridge::atomic_bridge_initiator` and `module atomic_bridge::atomic_bridge_counterparty`

### CODE REVIEW

File: protocol-units/bridge/move-modules/sources/MOVE.move [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+            constructor_ref,
+            option::some(withdraw),
+            option::some(deposit),
+            option::none(),
+        );
+    }
+
+    /// Allow a spender to transfer tokens from the owner's account given their signed approval.
+    /// Caller needs to provide the from account's scheme and public key which can be gotten via the Aptos SDK.
+    public fun transfer_from(
```
Comment (andygolay) [inline protocol-units/bridge/move-modules/sources/MOVE.move:171]: Might as well get rid of `transfer_from` once and for all?

**Review by andygolay — CHANGES_REQUESTED** [review]
This looks good, all tests pass. 

Logically because you substituted `move_fa` in for `moveth` throughout the bridge modules, and didn't add any new features to the asset module, it seems good. 

Do we want to create separate bridge modules for swapping ETH to and from MovETH? Or is it fine to replace them?

I requested removing `transfer_from` as we've established in previous discussions that it is unnecessary.

**Comment by 0xmovses** [comment]
I think it's fine to replace them for now.

File: protocol-units/bridge/move-modules/sources/MOVE.move (hunk 1 above)
Comment (0xmovses) [reply protocol-units/bridge/move-modules/sources/MOVE.move]: Removed `transfer_from` and re-ran the tests, they do run on CI too.

**Review by andygolay — APPROVED** [review]
:shipit:

**Pull request merged**